components/inputs/input-number.js

```diff
@@ -371,7 +371,10 @@ class InputNumber extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(Lit
 		});
 	}
 
-	focus() {
+	async focus() {
+		await this.updateComplete;
+		const inputTextElem = this.shadowRoot.querySelector('d2l-input-text');
+		await inputTextElem.updateComplete;
 		const elem = this.shadowRoot.querySelector('d2l-input-text');
 		if (elem) elem.focus();
```

Comment:
A pattern we've used elsewhere (like input-text) is this:

```suggestion
		const elem = this.shadowRoot.querySelector('d2l-input-text');
		if (elem) {
			elem.focus();
		} else {
			await this.updateComplete;
			this.focus();
		}
```

Reply:
Make sense, will test it out!